packages/imagemagick.rb

```diff
@@ -4,8 +4,8 @@ class Imagemagick < Package
   description 'Use ImageMagick to create, edit, compose, or convert bitmap images.'
   homepage 'http://www.imagemagick.org/script/index.php'
   version '7.0.7-9'
```

Comment:
Thank you for the update @rumpaar1.  Can you please bump the version to match the source?

Reply:
That would be what I forgot to do. Done now.